docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,344 @@ redirect_from:
+<!--TODO - Update image to match the latest App Directory sidebar (but remove the Workflows beta) -->
+![Basic Information page](../assets/basic-information-page.png "Basic Information page")
+
+Look around, add an app icon and description, and then let's start configuring your app 🔩
+
+---
+
+### Tokens and installing apps
+Slack apps use [OAuth to manage access to Slack's APIs](https://api.slack.com/docs/oauth). When an app is installed, you'll receive a token that the app can use to call API methods. 
+
+There are two token types available to a Slack app: user (`xoxp`) and bot (`xoxb`) tokens. User tokens allow you to call API methods on behalf of users after they install or authenticate the app. There may be several user tokens for a single workspace. Bot tokens are associated with bot users, and are only granted once in a workspace where someone installs the app. The bot token your app uses will be the same no matter which user performed the installation. Bot tokens are the token type that _most_ apps use.
+
+For brevity, we're going to use bot tokens for this guide.
+
+Navigate to the **OAuth & Permissions** on the left sidebar and scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**.
+
+For now, we'll just add one scope: [`chat:write`](https://api.slack.com/scopes/chat:write). This grants your app the permission to post messages in channels it's a member of.
+
+Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**. You'll be led through Slack's OAuth UI, where you should allow your app to be installed to your development workspace.
+
+Once you authorize the installation, you'll land on the **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
+
+<!--TODO - Update image to match the latest App Directory sidebar (but remove the Workflows beta) -->
+![OAuth Tokens](../assets/bot-token.png "Bot OAuth Token")
+
+> 💡 Treat your token like a password and [keep it safe](https://api.slack.com/docs/oauth-safety). Your app uses it to post and retrieve information from Slack workspaces.
+
+---
+
+### Setting up your local project
+With the initial configuration handled, it's time to set up a new Bolt project. This is where you'll write the code that handles the logic for your app.
+
+If you don’t already have a project, let’s create a new one. Create an empty directory:
+
+```shell
+mkdir first-bolt-app
+cd first-bolt-app
+```
+
+Next, we recommend using a Python virtual environment to manage your project's dependencies. Create and activate a new virtual environment:
+
+<details>
+<summary markdown="0">
+<h4>Setting up a Python virtual environment</h4>
+</summary>
+
+TODO - This is where we explain how to installing a Python version manager, virtual environment, and adding it to gitignore.
+
+---
+</details>
+
+<!-- TODO - Details on installing Python version manager, virtual environment, adding it to the git ignore, etc -->
+<!-- TODO - Do we use python or python3? -->
```

Comment:
If a developer is not familiar with Python versions and is going to start with the system python, explicitly mentioning `python3` this way should be helpful. Also, if a developer is aware of the difference between v2 and v3, the developer can understand why it's `python3`.

Reply:
Thanks for the advice @seratch! This makes complete sense and I like the observation that experience Python developers will understand why `python3` was used. I'll keep this code snippet as `python3` because it is best universal solution.